xml_converter/generators/cpp_templates/parseabletemplate.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+{% if page == "Icon": %}
```

Comment:
these `#includes` should be dynamically generated based on what classes are used.